### DISCUSSION TITLE
🌱 Beans

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#1990] [Ideas]
**This Proposal is still in draft mode. Expect any values to change**

The purpose of this discussion is to explore the treasure island proposal, gather feedback and make improvements.

## Beans

The long and awaited magic beans are coming. These beans have the power to grow the long awaited mutant crops & discover prizes.

_Beta testing expected to start late January_

**Goal of Beans**

- Provide a sink for resources
- Alternate route for earning SFL, gathering resources and rare items
- An alternate path for players. Expand land or use resources to plant magic beans
- Alternate path to discover food and level your Bumpkin

## Exotic Shop

_Players can purchase magic beans at the Exotic Shop on Helios_

<img width="485" alt="Screen Shot 2023-01-06 at 2 23 13 pm" src="https://user-images.githubusercontent.com/11745561/210923568-3bee693b-eb57-4234-8b29-81f820005b77.png">

The main ingredients for these bean boxes are wood, stone, iron and gold. This is to encourage players to collect resources.

**Magic Bean**

- 10 Wood & 10 Stone
- 10 in Stock
- 2 days to grow

**Shiny Bean**

_Coming Soon_

**Golden Bean**

_Coming Soon_

## Planting a Bean

A bean functions similar to a normal crop, but a player must place it on a free space on their island. A Bean occupies 2x2 of space. 

_This is a subtle encouragement for players to expand their island so they have extra room to place their beans_

Once a player has purchased a bean box, they can go to their Chest and 'place on map'.

<img width="486" alt="Screen Shot 2023-01-06 at 2 25 56 pm" src="https://user-images.githubusercontent.com/11745561/210923927-199ee902-2af4-44a2-af89-392f48744a87.png">

A player can then place the bean on an unoccupied piece of land.

<img width="141" alt="Screen Shot 2023-01-06 at 2 26 26 pm" src="https://user-images.githubusercontent.com/11745561/210923996-4266cafb-07bc-4cb2-a6e0-67ff959278db.png">

Once a bean is planted, the player must wait for the Bean to fully grow before being harvested.

## What can you find from a Bean?

Players can a range of items in a bean including mutant crops (with utility), giant crops, exotic crops to sell for SFL + a range of rare decorations.

When Shiny Bean and Golden Bean are introduced, more SFTs and decorations will also become available.

| Mutant Crop  | Utility |
| ------------- | ------------- |
| Mutant Sunflower | +0.25 Sunflowers  |
| Mutant Potato   | +0.25 Potatoes  |
| Mutant Pumpkin   | +0.25 Pumpkins  |
| Cabbage Patch Baby   | Crops grow 10% faster  |
| Cabbage Patch Baby 2   | +0.25 across all crops  |

_Exotic Crops can be sold at the Exotic Shop. Prices below are based on supply as of 6th Jan 2023_

| Exotic Crop  | SFL Sale Price |
| ------------- | ------------- |
| Golden Crop | 125 SFL  |
| Black Magic   | 62.5 SFL  |
| Golden Helios   | 25 SFL  |
| Chiogga   | 10 SFL  |
| Purple Cauliflower   | 3.75 SFL  |
| Adirondack Potato   | 1.875 SFL  |
| Warty Goblin Pumpkin  | 1.25 SFL  |
| White Carrot  | 0.625 SFL  |

| Decoration  | 
| ------------- | 
| Crop Bear | 
| Potato Bear | 
| Pumpkin Bear | 
| Giant Parsnip |
| Giant Radish |
| Giant Pumpkin | 
| Giant Potato |
| Potted Potato |
| Potted Pumpkin |
| Potted Cauliflower |
| Potted Radish |

## Carrot Sword

If a player has placed a carrot sword on their island, they will have a 200% increased chance of finding a Mutant Crop 🚀

**Comment by KUMARGFSTEALER** [comment]
Exotic Crops are not considered decorations? This means we can't put it on our farm?

**Reply by adamhannigan** [reply]
Initially they won't be placeable. If players desire it, we can definitely make them decorations

**Comment by mGranTY** [comment]
@adamhannigan the Sunflower Bear is already in-game, its a reward from the Sunflower SuperStar achievement.

**Reply by adamhannigan** [reply]
Good catch, we will provide a different variant of the bear 👍

**Comment by NomiceBinocle** [comment]
hey mose very interesting day i love what i read

but then a quick question: what will be the effect of the carrot sword on the magic beans? (from the start it must increase the chance of dropping mutant crops)

**Reply by adamhannigan** [reply]
Yes, that is correct. I will update the proposal

**Reply by NomiceBinocle** [reply]
Thanks ❤️😃

**Reply by MrSnorch** [reply]
> Yes, that is correct. I will update the proposal

When I bought the sword it had a 300% boost to mutant crops, why now 200%?

**Reply by adamhannigan** [reply]
@MrSnorch My bad, it should be whatever the previous boost was.

**Comment by KUMARGFSTEALER** [comment]
Since Cabbage Patch Baby 2 = +0.25 across all crops
does this mean that 
Cabbage Patch Baby = Crops grow 10% faster (is applicable for ALL crops too or just cabbage?)

**Reply by adamhannigan** [reply]
Currently, all crops 👍

**Comment by MrSnorch** [comment]
I also want to add that legacy skills increased the chance of a mutant crops👇

![IMG_20230109_025422](https://user-images.githubusercontent.com/104712725/211227480-77e4fa8a-f97c-4180-a718-810cbd1bef1c.jpg)
![IMG_20230109_025432](https://user-images.githubusercontent.com/104712725/211227482-d49d3fd6-806d-48f6-b515-fd3372c44aa1.jpg)

**Reply by adamhannigan** [reply]
Yes 👍

**Reply by alphacruze** [reply]
Will those legacy skills increase mutant crop chance? or no longer effective?

**Reply by VZozul** [reply]
Maybe enough? You know, I entered the game later, there were no more places in the beta and I was not advised to buy land on the OS, but just wait. And what do I have now? Not only did you get the best SFTs, which we new players are now buying at crazy prices. Do you still want your boosts to always work? Did you earn little? How long can this go on, how can I want to play and invest something, after I constantly read about your boosts in the chat in Discord. You talk about attracting new players, but you yourself are killing a fair and equal game! Okay, I came in later, I bought a Carrot Sword, which you got for a penny, but tying Legacy Skill here is too much! Soon with such an attitude, you will be selling SFTs between those who remained, which in principle is already happening. Maybe you should think about what is wrong, since there are so few players left? Look at mint fruit SFTs, a good indicator. Maybe you should start changing something? How would anyone want to play and invest money here, knowing that someone will have an eternal advantage. You support old players, do not let new ones be on a par, with whom will you play? On the contrary, you need to start supporting new players. Why give you bonuses to Legacy Skill even more? There were 152 thousand of you, now 1000 of you are actively playing at most. Do you want more bonuses, to whom will you sell these SFTs? You made money on Carrot Sword by selling it to newcomers, and you want to earn more on SFT with Magic Beam, which you will sell to yourself to each other, and then you are unlikely to sell it, because soon there will be only those who can get these SFTs for themselves, because every newcomer will look at it all and go looking for another game. Well, there was one very good indicator of movement in this direction, Iron Idol .. but there were hints in the chat that Vip received information about the need to save loot. And if this is so, I'm not 100% sure of this, but there were hints, I'm attaching a screen
![INS](https://user-images.githubusercontent.com/103659539/221430323-92457b3a-cde2-48e0-8b73-790c3e73e451.jpg)
And that's not all, it's just hard to find all the messages in the chat that hint at the fact that many people knew about leaving loot from TI. It’s not very nice in relation to all the newcomers, I’ll even say it’s low, and if I knew 100% that it was, I would immediately started selling everything and would have left the game. While the newcomers were selling loot in order to at least make some money, and not for withdrawal, but to support in-game purchases, there were people who knew, seriously? And with this approach you want new players? It is in your interests to make the game as fair for the new ones, so that they want to play, and so that you have something to sell to someone. Do you know how to do it? With the introduction of the new skill system, cancel all Legacy Skills and give everyone equal rights. That will be honest, you have already earned enough on your own, everything has its limit. Otherwise, the game will just die soon.

**Reply by alphacruze** [reply]
Legacy skills is a sign of having participated in the Beta, and those skills took a lot of effort to maintain, literal months of playing hence they still in effect

With a lot of your other stuff, the team has been actively working on things for new players and changing the approach on things, the Iron Doll is actually a pretty good example

It's random, with some of the requirements coming from "common" loot, not rare or ultra rare(whatever it's called)

Honestly if future drops continue with this kinda of methodology, there would be a lot of room for newer players to enter the game.

This is things I have said many times in Discord. I remember a mod or community member basically shouting at me going "So you saying that all boosts should have unlimited supply" and of course I answered "Yes, with some restrictions, AS that is HOW you bring in new players, not by having every single boost be limited to 1000 supply"

**Comment by Polysemouse** [comment]
Is the golden crop the same SFT as the one used in the 100 USDC golden crop event?

Also perhaps a better name can be given instead of attaching a number at the end for "Cabbage Patch Baby 2".

**Reply by adamhannigan** [reply]
Yes, we will reuse that since we already have it in our system 👍 Otherwise it would be a legacy token that sits around for awhile 😢

**Comment by buenti** [comment]
Hi Adam.. the mutants crop boost will last forever? Like the mutants chicken?

**Reply by adamhannigan** [reply]
Yes 👍

**Comment by maxjagaapp** [comment]
What the difference between the magic, shiny and golden? Drop different item? Or highest chances to get the best drop?

**Comment by speedtestl12** [comment]
@adamhannigan 
Tomorrow magical beans or is he going to be delayed as usual ?
You must respect your deadline.
You always can't meet your deadline.

**Reply by veenhdek** [reply]
I'm looking for this deadline could you point me toward it? Also afaik this team has exceeded every deadline they've set (except when they changed their new feature model and had deadlines that reflected the old system that did not include this proposal stage)

**Reply by adamhannigan** [reply]
@speedtestl12 Great question!

As per our live streams and Discord broadcasts over the past 3 weeks we explained that we would first be focussing on Fruits (which was released on time) and then Treasure.

As per our docs we are aiming to start work and testing on this late January/early February.

We've got a lot of features coming out in the mean time, so plenty to be excited about 😄

**Reply by adamhannigan** [reply]
I see your confusion, since the info graphic included the "16th of January date".

We assumed that the text added around this image would explain the actual date, but it appears that people may be skipping and only looking at the images.

I will remove the image to prevent any further confusion

**Comment by ghost** [comment]
Okay, when will it be in the game?

**Comment by Manu-M123** [comment]
Giant Radish
![gigan bet300-0001](https://user-images.githubusercontent.com/120758758/213842773-5a7b0a1e-2b91-4e7c-892c-79d2bebe9329.png)
![gigan bet bitrina 300-0001](https://user-images.githubusercontent.com/120758758/213842780-e3b21cdc-5e5a-4aaf-aae0-555665f3ee0d.png)
![gigan bet-0001](https://user-images.githubusercontent.com/120758758/213842787-e8f6accf-cc84-41b6-b8a1-b236e5830a99.png)
![gigan bet bitrina-0001](https://user-images.githubusercontent.com/120758758/213842790-6b273a7d-d923-4c83-8ce0-619161621624.png)
 Giant Pumpkin
![gigan betgigant calabaza 300-0001](https://user-images.githubusercontent.com/120758758/213842918-55b899b7-fd28-4661-af31-ac1f97da8af5.png)

![gigan betgigant calabaza a300 00-0001](https://user-images.githubusercontent.com/120758758/213842800-73a46ffc-7a08-463b-95a4-c86e07db1640.png)
d2c9f.png)
![gigan betgigant calabaza a100 00-0001](https://user-images.githubusercontent.com/120758758/213842803-26c14592-5ea7-494e-af36-7409280e549e.png)
![gigan betgigant calabaza300 00-0001](https://user-images.githubusercontent.com/120758758/213842806-ab75cc9d-f612-4cfb-b213-fb182c2f9237.png)
 Black Magic
![gigan betgisunflower black 1 62500 00-0001](https://user-images.githubusercontent.com/120758758/213842817-4cec5272-4826-4e8d-aedc-3a9852a926d9.png)
![gigan betgisunflower black 1 100 00-0001](https://user-images.githubusercontent.com/120758758/213842819-7c4809d9-1a26-4847-963a-47b6be6e0bf4.png)
2
![gigan betgisunflower sunflower blaxk 2 00-0001](https://user-images.githubusercontent.com/120758758/213842822-35309615-3af4-40f0-8033-aa28ed25b28d.png)
![gigan betgisunflower sunflower blaxasdasda 2 00-0001](https://user-images.githubusercontent.com/120758758/213842827-0101749a-8073-4a8f-a70a-723d21593aa6.png)
Golden Helios
![gigan betgisunflower sunflower blagolden sadasd0001](https://user-images.githubusercontent.com/120758758/213842846-0c211190-0924-4a6b-aef2-6416a84fc303.png)
![gigan betgisunflower sunflower blagolden helios 0001](https://user-images.githubusercontent.com/120758758/213842849-cc3cb993-1e0d-44fc-9992-0bd636c6cd2a.png)

ID:78922
ds: MANU m#4728

**Comment by blastaman** [comment]
Is this still thing or has it been scrapped?
